### PR TITLE
Add AUR package

### LINKED ISSUE #16: Create AUR Pkg
Would be nice to have an AUR Pkg

### PR DESCRIPTION
**Pull request by stewi1014**
Adds an Arch image and link to the AUR package

Closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Installation
 Install binary package for your GNU/Linux distribution:
 
 [![Binary packages](doc/distros.svg)](https://software.opensuse.org//download.html?project=home%3Akernc%3Axsuspender&package=xsuspender)
-
+[![Binary packages](doc/arch.svg)](https://aur.archlinux.org/packages/xsuspender-git/)
```
Comment (kernc) [inline README.md:56]: Let's rename these image alts as `Debian, Ubuntu` and `Arch Linux`.

Comment (kernc) [reply README.md:56]: And keep two lines blank before the next section.

Comment (stewi1014) [reply README.md:56]: Newline added, image file renamed.

Did you want to use a different naming convention for the images or is this good?

Comment (kernc) [reply README.md:56]: I meant image `alt=` text for a11y/robot navigation:
```html
<img src="doc/debian_ubuntu.svg" alt="Debian / Ubuntu">
```

Comment (stewi1014) [reply README.md]: Ah of course! Image names are updated.

**Comment by kernc** [comment]
Perfect, thanks! :cake:

**Pull request merged**